### PR TITLE
standardise text colors, and expose tooltip variables

### PR DESCRIPTION
**Pull request by taijuten**
#### What does this PR do? (please provide any background)
This fixes several issues within the SB Brands Project (https://github.com/orgs/holidayextras/projects/1)

* Bootstrap tooltips need to use brand colours
* Inconsistent margins on Peppa lists
* text-muted and text-muted-inverse need standardising on Peppa and Paultons

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?
Nay

#### Screenshots / Screencast
screen shot 2017-03-03 at 12 27 26

#### What gif best describes how you feel about this work?

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><hr><h3>Application brand dependencies</h3><small><strong>A list of applications and whether or not they will automatically absorb your changes</strong></small></br><small>You're wanting to deploy <strong>4.0.0</strong></small><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^3.0.0</td><td>❌</td></tr><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^3.0.0</td><td>❌</td></tr><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^3.0.0</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/globals/src/less/variables.less [hunk 1]
```diff
@@ -86,6 +86,22 @@
...
+//** Tooltip text color
+@tooltip-color:               #fff;
+//** Tooltip background color
+@tooltip-bg:                  #000;
+@tooltip-opacity:             .9;
+
+//** Tooltip arrow width
+@tooltip-arrow-width:         5px;
+//** Tooltip arrow color
+@tooltip-arrow-color:         @tooltip-bg;
```
Comment (nickine) [inline brands/globals/src/less/variables.less]: this whole block can come out of globals as it's already being set in bootstrap

File: brands/globals/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@holidayextras/brand-globals",
-  "version": "3.0.1",
+  "version": "3.0.2",
```
Comment (nickine) [inline brands/globals/package.json]: as you don't need the globals variables change - a v bump is unneeded

File: brands/paultons/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@holidayextras/brand-paultons",
-  "version": "0.0.9",
+  "version": "0.0.10",
```
Comment (nickine) [inline brands/paultons/package.json]: 🇹🇭

File: brands/paultons/src/less/variables.less [hunk 4]
```diff
@@ -10,12 +10,6 @@
 // --------------------------------------------------
 
 @body-bg:                  @brand-blue;
-@text-color:               @color-white;
-@text-color-inverse:       @gray-darker;
-@text-muted:               lighten(@text-color, 15%);
-@text-inverse:             @gray-darker;
-@headings-color-inverse:   @gray-darker;
-@text-muted-inverse:       lighten(@text-color-inverse, 15%);
```
Comment (nickine) [inline brands/paultons/src/less/variables.less]: why the deletion of these

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><hr><h3>Application brand dependencies</h3><small><strong>A list of applications and whether or not they will automatically absorb your changes</strong></small></br><small>You're wanting to deploy <strong>4.0.0</strong></small><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^3.0.0</td><td>❌</td></tr><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^3.0.0</td><td>❌</td></tr><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^3.0.0</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/peppa/src/less/lists.less [hunk 5]
```diff
@@ -1,7 +1 @@
 @import '../../../brand-globals/src/less/lists';
-
-.list-unstyled {
-  img, svg {
-    margin-right: 10px;
-  }
-}
```
Comment (nickine) [inline brands/peppa/src/less/lists.less]: is is for the usp icons and basket stuff? Not wanted?

File: brands/paultons/src/less/variables.less (hunk 4 above)
Comment (taijuten) [inline brands/paultons/src/less/variables.less]: `text-muted` is deprecated, and the others are specified below?

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><hr><h3>Application brand dependencies</h3><small><strong>A list of applications and whether or not they will automatically absorb your changes</strong></small></br><small>You're wanting to deploy <strong>4.0.0</strong></small><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^3.0.0</td><td>❌</td></tr><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^3.0.0</td><td>❌</td></tr><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^4.0.2</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^3.0.0</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/peppa/src/less/lists.less (hunk 5 above)
Comment (taijuten) [inline brands/peppa/src/less/lists.less]: With this, it was either remove it in here and make it part of the app-styles, or add it to the other brands.

I err'd towards removing from brands, as it feels like an app-specific issue. However, I'm happy to move it back into brands, and apply it to Paultons / Pottter too

Comment (nickine) [inline brands/peppa/src/less/lists.less]: I see.  We can move this into the app specific files for basket on PL.

**Pull request merged**